src/Stack/Stack.test.jsx

```diff
@@ -27,6 +27,14 @@ describe('<Stack />', () => {
       const wrapper = mount(<Stack gap={gap}>Content</Stack>);
       expect(wrapper.find('.pgn__vstack').hasClass(`pgn__stack-gap--${gap}`)).toEqual(true);
     });
+    it('renders reversed children with the vertical direction', () => {
+      const wrapper = mount(<Stack reversed>Content</Stack>);
+      expect(wrapper.find('.pgn__vstack').hasClass('pgn__stack-reversed')).toEqual(true);
+    });
+    it('renders reversed children with the horizontal direction', () => {
+      const wrapper = mount(<Stack direction="horizontal" reversed>Content</Stack>);
+      expect(wrapper.find('.pgn__hstack').hasClass('pgn__stack-reversed')).toEqual(true);
+    });
```

Comment:
We shouldn't be using `enzyme` in the tests anymore, because of #2581, even though it's not merged yet. Please switch these new tests to react-testing-library and avoid checking for class, it's an anti-pattern in react-testing-library, you should instead pass a couple of elements as children to the `Stack` (instead of just `Content` text) and check that they appear in the correct order in DOM.

Reply:
Ooops, didn't think about this, we can leave it like this then.